cafy_pytest/plugin.py

```diff
@@ -209,6 +211,10 @@ def pytest_addoption(parser):
             metavar='mongo_mode',
             help='Variable to enable mongo read/write, default is None')
 
+    group.addoption('--collection', action='store', dest='collection',
+            type=str, default=None,
```

Comment:
Do u want to make this type as list ? If not, then modify the help statement accordingly